src/Microsoft.VisualStudio.Validation/Verify.cs

```diff
@@ -126,9 +126,11 @@ public static void NotDisposed(IDisposableObservable disposedValue, string messa
         [DebuggerStepThrough]
         public static void NotDisposed([DoesNotReturnIf(false)] bool condition, object disposedValue, string message = null)
         {
+            Requires.NotNull(disposedValue, nameof(disposedValue));
```

Comment:
`public static void NotDisposed(IDisposableObservable disposedValue, string message = null)` requires `disposedValue` not null.
And we already has `public static void NotDisposed([DoesNotReturnIf(false)]bool condition, string message)`
So I think we should requires `disposedValue` not null here.

Reply:
Seems sensible, though a breaking change.

If accepted, there should be a test case that validates the ANE.

Reply:
That's a breaking change for a library that's used in a *lot* of places. I don't recall the reasoning for the difference in accepted `null` here, if there even was one. But this isn't one of those breaking changes that seem valuable to make in the face of possible costs. So I'm reverting this part of your change prior to merging.